src/client/js/Widgets/DiagramDesigner/DiagramDesignerWidget.js

```diff
@@ -359,6 +359,7 @@ define([
     DiagramDesignerWidget.prototype.destroy = function () {
         this.__loader.destroy();
         this._removeToolbarItems();
+        this.connectionRouteManager.destroy();
         //TODO: what about item and connection destroys????
```

Comment:
According to git blame, Robert added that line May 14, 2013... Probably should leave it for now